content/api/rest-sdk.textile

```diff
@@ -0,0 +1,242 @@
+---
+title: Constructor
```

Comment:
I think we should add a meta description and keywords after you added it to all the pages.

Reply:
Agreed. Done.